librz/core/core_private.h

```diff
@@ -96,6 +96,12 @@ RZ_IPI void rz_core_agraph_print_json(RzCore *core);
 RZ_IPI void rz_core_agraph_print_gml(RzCore *core);
 RZ_IPI void rz_core_agraph_print_write(RzCore *core, const char *filename);
 
+/* cbin.c */
+RZ_IPI void rz_core_bin_entries_print(RzCore *core, RzCmdStateOutput *state);
```

Comment:
@thestr4ng3r @XVilka @wargio  not sure about these API... shall they get the RzBinFile as argument or shall they just work on the current binfile?

Reply:
But there are not APIs, they are internal. Personally, I think it's fine. Lets see what others think, especially @thestr4ng3r

Reply:
Probably many of these can just be removed and directly implemented in the cmd_handler in cmd_info.c, what do you think? They don't do much in terms of API..

Reply:
I am completely fine with this proposition. 👍 It makes an absolute sense, especially if not used anywhere else.

Reply:
Improved the situation here. Marking as resolved.